pennylane_lightning/core/_state_vector_base.py

```diff
@@ -128,7 +136,10 @@ def _apply_basis_state(self, state, wires):
             raise ValueError("BasisState parameter and wires must be of equal length.")
 
         # Return a computational basis state over all wires.
-        self._qubit_state.setBasisState(list(state), list(wires))
+        if use_async is None:
+            self._qubit_state.setBasisState(list(state), list(wires))
+        else:
+            self._qubit_state.setBasisState(list(state), list(wires), use_async)
```

Comment:
Do we really need to need `use_async` ? I think everything in L-GPU is sync.

Reply:
Yes. At the end of the implementation, I was thinking the same. Do you suggest removing it? 
I am not sure but do you think that someone will use this option? :thinking:

Reply:
Done! I remove the `use_async` as optional arg in the binding layer  :rocket: